week5.1-ex2.py

```diff
@@ -0,0 +1,77 @@
+from calendar import calendar
+import random
+import datetime
+
+dictionary_of_months = {'01': 31, '02': 28, '03': 31, '04': 30, '05': 31, '06': 30, '07': 31, '08': 31, '09': 30,
+                        '10': 31, '11': 30, '12': 31}
+
+
+def checks_year(year):
```

Comment:
This is a nice cpp code, in python we are lazy, so we have pre-defined funtions do to our work for us. Use them, and remember to always think - does this have an existing implementation already  ? 
https://www.programiz.com/python-programming/datetime/strptime

Reply:
https://realpython.com/python-datetime/ read this as well - a module that helps to handle dates